dev/tc/datarepo-api.yaml

```diff
@@ -0,0 +1,25 @@
+---
+image:
+  tag: 1.38.0
+env:
+  GOOGLE_PROJECTID: broad-jade-dev
```

Comment:
This is what we switched during the call - If we get devops to create TC a project, then this can be switched to broad-jade-tc.